examples/plot_fmri_data_example.py

```diff
@@ -1,314 +0,0 @@
-"""
-Support recovery on fMRI data
-=============================
-
-This example compares several methods that estimate a decoder map support
-with statistical guarantees. More precisely, we aim at thresholding the
-weights of some estimated decoder maps according to the confidence we have
-that they are nonzero. Here, we work with the Haxby dataset and we focus on
-the 'face vs house' contrast. Thus, we consider the labeled activation maps
-of a given subject and try produce a brain map that corresponds to the
-discriminative pattern that makes the decoding of the two conditions.
-
-In this example, we show that standard statistical methods (i.e., method
-such as thresholding by permutation test the SVR or Ridge decoder or the
-algorithm introduced by :footcite:t:`gaonkar_deriving_2012`) are not powerful when applied on
-the uncompressed problem (i.e., the orignal problem in which the activation
-maps are not reduced using compression techniques such as parcellation).
-This is notably due to the high dimensionality (too many voxels) and
-structure of the data (too much correlation between neighboring voxels).
-We also present two methods that offer statistical guarantees but
-with a (small) spatial tolerance on the shape of the support:
-clustered desparsified lasso (CLuDL) combines clustering (parcellation)
-and statistical inference ; ensemble of clustered desparsified lasso (EnCluDL)
-adds a randomization step over the choice of clustering.
-
-EnCluDL is powerful and does not depend on a unique clustering choice.
-As shown in :footcite:t:`chevalier2021decoding`, for several tasks the estimated
-support (predictive regions) looks relevant.
-
-References
-----------
-.. footbibliography::
-
-"""
```

Comment:
Please don't remove the example altogether ! Some parts are worth keeping ...

Reply:
Do you want to keep an example with only EnCluDl and CluDL?

Reply:
Yes

Reply:
I put them back.